jwst/extract_2d/nirspec.py

```diff
@@ -37,7 +37,7 @@ def nrs_extract2d(input_model, slit_name=None, apply_wavecorr=False, reference_f
     exp_type = input_model.meta.exposure.type.upper()
 
     wavecorr_supported_modes = ['NRS_FIXEDSLIT', 'NRS_MSASPEC', 'NRS_BRIGHTOBJ',
-                                'NRS_AUTOFLAT']
+                                'NRS_AUTOFLAT', 'NRS_AUTOWAVE']
```

Comment:
Further down at line 69 the code checks if exp_type is `BRIGHTOBJ`. Should it check also for `NRS_AUTOWAVE` and `LAMP_MODE=BRIGHTOBJ`?

Reply:
See comment above